frontend/src/components/common/form/StepForm/SteppedForm.tsx

```diff
@@ -92,11 +99,18 @@ export const SteppedForm = function<T extends object = {}>({
   if (!getTabs) {
     initialValues.tabs = [{ activeStep: initialValues.activeStep }];
   }
+  if (!initialValues.tabs && !!getTabs) {
+    initialValues = setIn(
+      initialValues,
+      'tabs',
+      getTabs(initialValues.data).map(t => ({ activeStep: 0, name: t })),
+    );
+  }
 
   const tabTitle = (title: string, index: number) => {
     return (
       <>
-        <p>{title}</p>
+        {title.length < 20 ? <p>{title}</p> : <abbr title={title}>{title.substr(0, 20)}</abbr>}
```

Comment:
if the name is too long replace with abbreviation.

Reply:
May make this into a function/util later this sprint.

Reply:
oh that's really handy

Reply:
`<p>{title?.substr(0, 20)}</p>` should work and you don't need to have so much typing.

Reply:
That doesn't provide the same functionality. I want the title to display normally if it below the max length, or as an abbr if it is greater then the max length (some pictures for context):

![image](https://user-images.githubusercontent.com/41091511/101820717-db835c00-3adb-11eb-9dd3-89021d373fe9.png)

the first tab is an abbr, the second tab is a p.

Reply:
If it's less than `20` then `title.substr(0, 20)` should result in the same thing I believe.